### DISCUSSION TITLE
How to add time to a saved name?

### DISCUSSION THREAD

**Discussion by bayunhuiye** [NickeManarin/ScreenToGif#1199] [Q&A] [answered]
How to add time to a saved name?
Similar to what is shown in the following image
![image](https://github.com/NickeManarin/ScreenToGif/assets/20291630/fd7465bb-306f-467d-85c2-81ba9a2ce004)

**Comment by NickeManarin** [comment, marked as answer]
You need to add a date/time format inside `? ?` (question marks) such as:

`Animation ?hh-mm-ss-fff?` = `Animation 09-19-20-453`

Theres a table [in this link](https://learn.microsoft.com/en-us/dotnet/standard/base-types/custom-date-and-time-format-strings) to all custom format characters.

**Reply by NickeManarin** [reply]
For your example, it would be `?d-yyyy-M-h-mm?` I believe.

**Reply by bayunhuiye** [reply]
Thank you, this method is very useful.
